WebRoot/js/zimbraMail/share/view/dialog/ZmUploadDialog.js

```diff
@@ -55,6 +55,7 @@ ZmUploadDialog.prototype.constructor = ZmUploadDialog;
 
 ZmUploadDialog.UPLOAD_FIELD_NAME = "uploadFile";
 ZmUploadDialog.UPLOAD_TITLE_FIELD_NAME = "uploadFileTitle";
+ZmUploadDialog.UPLOAD_INPUT_LABEL = "ZmUploadFileLabel";
```

Comment:
Minor: Can we update property name to indicate that it's `class` for label. I thought initially that it's text of label. 
Probably: `UPLOAD_LABEL_CLASS `